src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -1781,7 +1781,7 @@ impl Step for Extended {
                 .arg(exe.join("CargoGroup.wxs"))
                 .arg("-t")
                 .arg(etc.join("msi/remove-duplicates.xsl"))
-                .run(builder);
+                .run(builder.context());
```

Comment:
As a shortcut, could we make `run` take `AsRef<ExecutionContext>` and implement it for `Builder`? It seems like needless noise to call this method everywhere, when 90% of the time you just have a builder that wraps the context.